examples/demo-atmos.d/atmos.yaml

```
@@ -0,0 +1,41 @@
+  # Absolute path
+  #base_path: "vendor.d/vendor1.yaml"
+
+logs:
+  file: "/dev/stderr"
+  level: Info
+
+# Custom CLI commands
+
+# No arguments or flags are required
+commands:
+- name: "test"
```

yamllint (1.29.0-1)

[warning] wrong indentation: expected 2 but found 0

(indentation)